aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -339,6 +338,12 @@ void resize_tensor(Tensor* output) {
   return [mpsGraph castTensor:tensor toType:getMPSScalarType(toType) name:@"castTensor"];
 }
 
+MPSGraphTensor* convertNHWCtoNCHW(MPSGraph *mpsGraph, MPSGraphTensor* tensor) {
+  // TODO: use transposeTensor(permutation:) to avoid passing negative dims to NSUInteger args
+  return [mpsGraph transposeTensor:[mpsGraph transposeTensor:tensor dimension:-1 withDimension:-2 name:nil]
```

Comment:
If negative dims are not working (or the variables are NSUInteger), you should be able to pass directly the wrapped dimensions:

```suggestion
  return [mpsGraph transposeTensor:[mpsGraph transposeTensor:tensor dimension:3 withDimension:2 name:nil]
                                                              dimension:2 withDimension:1 name: nil];
```